api/src/main/java/org/pmiops/workbench/cdr/CdrVersionService.java

```diff
@@ -97,27 +101,69 @@ public DbCdrVersion findAndSetCdrVersion(Long cdrVersionId) {
     return dbCdrVersion;
   }
 
-  /**
-   * Retrieve all the CDR versions visible to users with the specified data access level. When
-   * {@link DataAccessLevel#PROTECTED} is provided, CDR versions for both {@link
-   * DataAccessLevel#REGISTERED} and {@link DataAccessLevel#PROTECTED} are returned. Note: this
-   * relies on {@link DbUser#dataAccessLevel} accurately reflecting that the user is in the
-   * authorization domain that has access to the CDR version BigQuery data sets with the matching
-   * {@link DataAccessLevel} values.
-   *
-   * @param dataAccessLevel the data access level of the user
-   * @return a list of {@link DbCdrVersion} in descending timestamp, data access level order.
-   */
-  public List<DbCdrVersion> findAuthorizedCdrVersions(DataAccessLevel dataAccessLevel) {
-    ImmutableSet<Short> visibleValues = DATA_ACCESS_LEVEL_TO_VISIBLE_VALUES.get(dataAccessLevel);
-    if (visibleValues == null) {
-      return ImmutableList.of();
+  public Optional<DbCdrVersion> findByCdrVersionId(Long cdrVersionId) {
+    return Optional.ofNullable(cdrVersionDao.findByCdrVersionId(cdrVersionId));
+  }
+
+  @Deprecated // only handles the Registered Tier
+  public CdrVersionListResponse getCdrVersions() {
```

Comment:
This implements the existing endpoint and provides (equivalently) existing behavior, with one exception: PROTECTED users are not handled.  That's OK because there are no PROTECTED users and no way for a user to get into that state.

Reply:
See the old version of CdrVersionsController in this PR for comparison